railties/lib/rails/application/configuration.rb

```diff
@@ -34,7 +34,10 @@ def initialize(*)
         @filter_redirect                         = []
         @helpers_paths                           = []
         if Rails.env.development?
-          if defined?(RubyVM) && RubyVM.respond_to?(:keep_script_lines=)
+          # In Ruby 3.2, error_highlight is able to handle errors in templates
+          # without `keep_script_lines`. In versions below 3.2 however,we need
+          # script lines to implement this.
+          if RUBY_VERSION > "3.2" && RubyVM.respond_to?(:keep_script_lines=)
```

Comment:
I may be misunderstanding, but should this be `< "3.2"`?

Reply:
The number of times I have messed up greater or less than is too high. Anyway we decided to drop keep script lines. 😄